e2e/pages/ShowPage.js

```diff
@@ -76,18 +76,17 @@ export default (url, initialField = 'title') => driver => ({
         return driver.sleep(200);
     },
 
-    delete() {
-        return driver
-            .findElement(this.elements.deleteButton)
-            .then(button => button.click())
-            .then(() =>
-                driver.findElement(this.elements.deleteConfirmButton).click()
-            )
-            .then(() => driver.sleep(200))
-            .then(() =>
-                driver.wait(until.elementLocated(this.elements.snackbar), 3000)
-            )
-            .then(() => driver.findElement(this.elements.body).click()) // dismiss notification
-            .then(() => driver.sleep(200)); // let the notification disappear (could block further submits)
+    async delete() {
+        const deleteButton = await driver.findElement(
+            this.elements.deleteButton
+        );
+        await deleteButton.click();
+
+        const deleteConfirmationButton = await driver.findElement(
+            this.elements.deleteConfirmButton
+        );
+        await deleteConfirmationButton.click();
+
+        await this.waitUntilDataLoaded();
```

Comment:
Just wait the list page has loaded instead of relying on a MaterialUI component. I guess their API changed.